chef/data_bags/crowbar/migrate/nova_dashboard/026_use_keystone_session_timeout.rb

```diff
@@ -0,0 +1,13 @@
+def upgrade ta, td, a, d
+  keystone_instance = a['keystone_instance']
+  proposal_obj = ProposalObject.find_proposal("keystone", keystone_instance)
+  keystone_timeout = (proposal_obj["attributes"]["keystone"]["token_expiration"] || 3600)/60
```

Comment:
Hrm, now I got your point. I guess in the end 14400 is more correct as it matches what we get after applying both updates. Though it will trigger https://bugzilla.suse.com/show_bug.cgi?id=941537 as long as the keystone update is not applied, but that is not really  a regression)

@mmnelemane can you change it once more? Sorry.